src/bramble/client/StateManager.js

```diff
@@ -139,6 +139,10 @@ define(function() {
                 get: function()  { return getBool(storage, "allowJavaScript"); },
                 set: function(v) { storage.setItem(prefix("allowJavaScript"), v); }
             },
+            allowAutoComplete: {
```

Comment:
I have a general concern.  In some places you use `Autocomplete` and in others `AutoComplete`.  Once upon a time I implemented a bunch of code in Firefox related to the FullScreen API which later became the Fullscreen API, and it left me very nervous about the bugs that can creep in with such inconsistencies.  When I Google this, most people seem to use "Autocomplete."  Should we do that everywhere?  I actually don't care as long as we're consistent in all places, here and in Thimble.